go/host/enclave/guardian.go

```diff
@@ -647,12 +644,23 @@ func (g *Guardian) periodicBundleSubmission() {
 				g.logger.Error("Unable to get bundle range from management contract", log.ErrKey, err)
 				continue
 			}
-			bundle, err := g.enclaveClient.ExportCrossChainData(context.Background(), from.Uint64(), to.Uint64())
+
+			if from.Uint64() > fromSequenceNumber {
+				fromSequenceNumber = from.Uint64()
+			}
+
+			bundle, err := g.enclaveClient.ExportCrossChainData(context.Background(), fromSequenceNumber, to.Uint64())
```

Comment:
is this excluding the "to" value?

Reply:
I think not, but if something gets published the from value will override as it gets returned +1